plugins/+ciplugins/+github/WorkflowCommandsPlugin.m

```diff
@@ -0,0 +1,22 @@
+classdef WorkflowCommandsPlugin < matlab.buildtool.plugins.BuildRunnerPlugin
```

Comment:
The name feels a little general. `WorkflowCommandsPlugin` could also be the name of a CircleCI plugin. This could also be considered a strength if we want to use the namespace as the differentiator and keep the names of similar plugins the same across CI platforms. We could also consider something like `GitHubLogPlugin`.

Reply:
The workflow in circle CI is different from `WorkflowCommands` in GitHub. this rather is more aligned to  what plugin does (which adds workflow command to log). for me this is much clearer what it does than `GithubLog` which is much generic. any  thoughts ?

Reply:
Maybe the name should be more about the intent of the plugin, rather than the particular implementation of how it is achieving that intent. The intent of this plugin is to make the runner better integrate with GitHub Actions. In particular, improve the runners log output to play better with GitHub Actions log. With that lens, `GitHubActionsLogPlugin` or `GitHubLogPlugin` or `GitHubActionsOutputPlugin` feel fitting.

Reply:
picking up `GitHubLogPlugin`